aw-datastore/src/datastore.rs

```diff
@@ -202,13 +204,42 @@ impl DatastoreInstance {
 
         let mut ds = DatastoreInstance {
             buckets_cache: HashMap::new(),
+            ignored_apps: vec![],
+            ignored_titles: vec![],
             first_init,
             db_version,
         };
         ds.get_stored_buckets(&conn)?;
+        ds.get_stored_ignores(&conn)?;
```

Comment:
So this is only done on startup? So if you ever change the list of apps/titles to ignore, you have to restart aw-server? Does not seem optimal.

Reply:
Yes, this is a prototype to see how well it works.   I'm using it great and my list never changes so far.   I suspect just like in @raudabaugh code modification the list is generated normally once and your good to go.   This list should rarely change...  However, I agree that if we add some UI in front, that somehow we need to trigger the reload of the values on its change...  

In addition I do believe Sam has a valid use case of rather than Ignoring the entries to use Redacted so this code should be enhanced to have Redacted_apps and Redacted_titles in addition to ignored as both are valid use cases.